gpt4all-chat/modellist.h

```diff
@@ -119,6 +122,7 @@ struct ModelInfo {
     QString m_filename;
     double  m_temperature          = 0.7;
     double  m_topP                 = 0.4;
+    double  m_minP                 = 0.1;
```

Comment:
kalomaze's suggestion is usually to use either min-p *or* top-p - not both. Although llama.cpp does default to top-p=.95 and min-p=.05, despite that advice.

Reply:
I recognize we are at the mercy of llama.cpp sequencing of the samplers and it makes sense to not use both. So should we do 0 by default and let people disable top-p on their own via 1.0 and increase min-p as desired?

Reply:
Made 0 to disable by default.